src/mixins/filing-template-mixin.ts

```diff
@@ -432,98 +436,90 @@ export default class FilingTemplateMixin extends Vue {
 
-    // Set Office Addresses
-    this.setOfficeAddresses(businessSnapshot[2])
+    // Store office addresses
+    // *** TODO: should restore office addresses from alteration, not snapshot???
+    this.setOfficeAddresses(businessSnapshot.incorporationAddress)
 
-    // Set Business Contact
-    const contact = {
+    // Store people and roles
+    // *** TODO: should restore people and roles from alteration, not snapshot???
+    this.setPeopleAndRoles(
+      businessSnapshot.orgPersons?.map(director => {
+        return {
+          officer: {
+            firstName: director.officer.firstName,
+            lastName: director.officer.lastName
+          },
+          mailingAddress: director.deliveryAddress,
+          deliveryAddress: director.mailingAddress,
+          roles: [
+            {
+              roleType: director.role in RoleTypes ? director.role
+                : Object.values(RoleTypes).find(role => {
+                  if (role.toLocaleLowerCase() === director.role.toLocaleLowerCase()) {
+                    return role
+                  }
+                }),
+              appointmentDate: director.appointmentDate,
+              cessationDate: null
+            }
+          ]
+        }
+      }) || []
+    )
+
+    // Store share classes and resolution dates
+    const shareStructure = filing.alteration.shareStructure
+    this.setShareClasses(shareStructure.shareClasses)
+    this.setResolutionDates(shareStructure.resolutionDates)
+    this.setOriginalResolutionDates(this.getSnapshotShareStructure.resolutionDates)
+
+    // Store business contact
+    this.setBusinessContact({
       ...filing.alteration.contactPoint,
       confirmEmail: filing.alteration.contactPoint.email
-    }
-    this.setBusinessContact(contact)
-
-    // Set People and Roles
-    this.setPeopleAndRoles(businessSnapshot[3].directors?.map(director => {
-      return {
-        officer: {
-          firstName: director.officer.firstName,
-          lastName: director.officer.lastName
-        },
-        mailingAddress: director.deliveryAddress,
-        deliveryAddress: director.mailingAddress,
-        roles: [
-          {
-            roleType: director.role in RoleTypes ? director.role
-              : Object.values(RoleTypes).find(role => {
-                if (role.toLocaleLowerCase() === director.role.toLocaleLowerCase()) {
-                  return role
-                }
-              }),
-            appointmentDate: director.appointmentDate,
-            cessationDate: null
-          }
-        ]
-      }
-    }))
-
-    // Set Share Structure
-    this.setPreviousResolutionDates(this.getSnapshotShareStructure.resolutionDates)
-    if (filing.alteration.shareStructure) {
-      this.setResolutionDates(filing.alteration.shareStructure.resolutionDates)
-      this.setShareClasses(filing.alteration.shareStructure.shareClasses)
-    } else {
-      // if it exists, load data from old schema
```

Comment:
I removed this code since we don't have any existing alterations using the old schema.